layers/+spacemacs/spacemacs-navigation/packages.el

```diff
@@ -34,12 +34,14 @@
     (progn
       (define-key spacemacs-buffer-mode-map "o" 'spacemacs/ace-buffer-links)
       (with-eval-after-load 'info
-        (define-key Info-mode-map "o" 'ace-link-info))
```

Comment:
Can you explain why you have replaced the mode specific functions with `ace-link`?

Reply:
Ah I forgot to set them back to original. I've read the docstring in `ace-link-man` and it says this function will open a link in `man` buffer. Obviously, when a link in a man page is e.g. some url then you do not want to open that link in a man buffer. (indeed the action function in `ace-link-man` calls actually `#'man-follow`) So I just experimented if maybe using `ace-link` would highlight all links instead of only man page links.

Anyway, the docstring for ace-link says it will just run the link function for the current major mode, so it also calls `ace-link-man` (and similar for the other modes). I am not sure if it is necessary to call the mode specific functions again. Maybe at some point ace-link will get some more powerful functionality than the mode specific functions (i.e. opening different style links in their corresponding mode buffers).

I even think that possibly `link-hint-open-link` will also work better for the other buffers. But that should be tested first then.

So in the end, you can decide if you prefer to keep the mode specific or the general `ace-link` functions.

Reply:
I will give it a quick test and change it back to the mode specific one if necessary. If I see no issues using the more generic function is ok with me.